Index: package/redhat/gfarm.spec
===================================================================
--- package/redhat/gfarm.spec	(revision 11182)
+++ package/redhat/gfarm.spec	(working copy)
@@ -266,7 +270,9 @@ sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_l
 sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
 fi
 
-make
+make -j 4 || exit 1
+cat /var/tmp/rpm-tmp.*
+exit 0
 
 %install
 make DESTDIR=${RPM_BUILD_ROOT} \
